hermes-console-vue/src/views/admin/constraints/create-constraint-form/CreateConstraintFormView.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+  import { Ref, ref } from 'vue';
+  import { subscriptionFqn } from '@/utils/subscription-utils/subscription-utils';
+  import type { Constraint } from '@/api/constraints';
+
+  const props = defineProps<{
+    subscription: boolean;
```

Comment:
WDYT about changing this props to 'isSubscription'? It bamboozled me for the first time, because I thought that You are passing the whole subscription object in here

Reply:
good idea, fixed